docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-FROM codercom/code-server:v2
+FROM codercom/code-server:v2.1699-vsc1.41.0
```

Comment:
this line should be `FROM codercom/code-server:2.1699-vsc1.41.0` you have a v before the version

Reply:
Fixed